src/context/AppProvider/AppContext.tsx

```diff
@@ -192,64 +190,121 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
           Object.assign(accountMetadataByAccountId, accountIdsAndMetadata)
 
           const { getAccount } = portfolioApi.endpoints
-          const accountPromises = accountIds.map(accountId =>
-            dispatch(getAccount.initiate({ accountId }, { forceRefetch: true })),
-          )
 
-          const accountResults = await Promise.allSettled(accountPromises)
+          // Chunks of AccountIds
+          // For UTXOs, each chain gets its own chunk to detect chain activity over other scriptTypes
+          // For others, no need to use chunks, but for the sake of consistency, we keep the same structure
+          const accountNumberAccountIdChunks = (
+            _accountIds: AccountId[],
+          ): Record<ChainId, AccountId[]> => {
```

Comment:
[suggestion]: Rename this to `accountNumberAccountIdsByChainId` or similar and update the rest of the vernacular to match.

These aren't really chunks - chunking typically refers to batching an array into multiple smaller arrays.

Reply:
Ab-solutely 💯: 5c094cc272